src/Parser/MarksmanshipHunter/Modules/Items/UllrsFeatherSnowshoes.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SpellUsable from 'Parser/Core/Modules/SpellUsable';
+
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+
+import { formatNumber } from "common/format";
+import SpellLink from "common/SpellLink";
+
+const COOLDOWN_REDUCTION_MS = 800;
+
+class UllrsFeatherSnowshoes extends Module {
+  static dependencies = {
+    combatants: Combatants,
+    spellUsable: SpellUsable,
+  };
+
+  effectiveTrueshotReductionMs = 0;
+  wastedTrueshotReductionMs = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasFeet(ITEMS.ULLRS_FEATHER_SNOWSHOES.id);
+  }
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.AIMED_SHOT.id && spellId !== SPELLS.MARKED_SHOT.id && spellId !== SPELLS.ARCANE_SHOT.id && spellId !== SPELLS.MULTISHOT.id) {
```

Comment:
Although it's quite clear what this condition does, it's uncomfortably long. What about this?
```jsx
// before class
const AFFECTED_ABILITIES = [
   SPELLS.AIMED_SHOT.id,
   SPELLS.MARKED_SHOT.id,
   SPELLS.ARCANE_SHOT.id,
   SPELLS.MULTISHOT.id,
];
.
.
.
if (AFFECTED_ABILITIES.every(id => spellId !== id)) {
...
}

```